doc/specifications/rest/REST-API-V2.rst

```diff
@@ -7403,7 +7403,10 @@ View XML Schema
 
       <xsd:complexType name="queryType">
         <xsd:all>
-          <xsd:element name="Criterion" type="criterionType" />
+          <!-- Criteria is deprecated since 6.6.0. Use 'Filter' instead. -->
+          <xsd:element name="Criteria" type="criterionType" />
```

Comment:
name change, should continue to `Criterion` no? _(and in API at least it is not actually plural, casue you typically assign a LogicalAnd or LogicalOr on it)_